op-acceptance-tests/tests/interop/proofs/proposer_test.go

```diff
@@ -0,0 +1,68 @@
+package proofs
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/wait"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/lmittmann/w3"
+)
+
+var (
+	gameCountFn        = w3.MustNewFunc("gameCount()", "uint256")
+	gameAtIndexFn      = w3.MustNewFunc("gameAtIndex(uint256)", "uint32, uint64, address")
+	rootClaimFn        = w3.MustNewFunc("rootClaim()", "bytes32")
+	l2SequenceNumberFn = w3.MustNewFunc("l2SequenceNumber()", "uint256")
+)
+
+func TestProposer(gt *testing.T) {
```

Comment:
The `interop/proofs` package needs to be added to [acceptance-tests.yaml](https://github.com/ethereum-optimism/optimism/blob/53ec05442b76c88369cabac0b194cc35546c5b4a/op-acceptance-tests/acceptance-tests.yaml#L43-L51) right?

Reply:
No because this test is already executed as part of the general `go-test` cci job.
The tests in acceptance-tests.yaml are executed on kurtosis, which isn't applicable here.